notebooks/default-prep-script.ipynb

```diff
@@ -312,7 +312,16 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "8",
-   "metadata": {},
+   "metadata": {
+    "nbmake": {
+     "post_cell_execute": [
+      "if sycamore.EXEC_LOCAL != sycamore.ExecMode.RAY:",
+      "    import sys",
+      "    # TODO: eric - remove the True",
+      "    assert True or 'ray' not in sys.modules"
```

Comment:
what does this do?

Reply:
Lets you add "tests" to notebooks. If we're running in local mode, then ray should not be in sys.modules.
I was planning on having the test script run the notebooks and then replace EXEC_LOCAL with EXEC_RAY and then running them a second time to make sure the notebooks work in both modes.  However, the check I wanted to add doesn't work yet.